Hymns/Home/HomeView.swift

```diff
@@ -1,17 +1,18 @@
+import Resolver
 import SwiftUI
 
 struct HomeView: View {
 
     @State var searchText: String = ""
     @ObservedObject private var viewModel: HomeViewModel
 
-    init(viewModel: HomeViewModel) {
+    init(viewModel: HomeViewModel = Resolver.resolve()) {
         self.viewModel = viewModel
     }
 
     var body: some View {
         VStack(alignment: .leading, spacing: 10) {
-            Text("Look up any hymn").customTitleLayout()
```

Comment:
Probably should also make this a CustomTitle to match with browse, favorites, and settings. It looks to me like on the android app they were all the same text sizing and even placement. The one that is hard to tell is this one the Look up any hymn. Is it the same font size and location. Hard to tell.

Reply:
I'm pretty sure they are the same font and size and placement. But I fix according to your comments to make it even more obvious/explicit. let me know if these address your comments